File: front/src/models/party.ts
import React from "react";

export interface Party {
    price : number|null;
    paymentDay: number|null;
    paymentMonth: number|null;
    userCount: number|null;
    maxUser: number|null;
    partyId: number|null;
    ottName: string|null;
    openChatUrl: string|null;
}

export const defaultData : Party ={
    price : null,
    paymentDay : null,
    paymentMonth: null,
    userCount: null,
    maxUser: null,
    partyId: null,
    ottName: "",
    openChatUrl: ""
}

File: server/src/main/java/com/fott/use_mybatis/domain/Party.java
package com.fott.use_mybatis.domain;


import com.sun.istack.internal.NotNull;
import lombok.*;

@NoArgsConstructor
@RequiredArgsConstructor
@Getter
@Setter
@ToString
public class Party {
    private Long partyId;

    @NotNull
    private Long ottId;

    @NotNull
    private Integer userCount;

    @NotNull
    private String openChatUrl;

    @NotNull
    private Integer paymentDay;

    @NotNull
    private Integer paymentMonth;
}
